### DISCUSSION TITLE
Need help installing Homebrew on macOS Ventura

### DISCUSSION THREAD

**Discussion by Skylamar** [Homebrew/discussions#4494] [Getting started]
### Output of `brew config`

```shell
Not sure, see Description
```

### Output of `brew doctor`

```shell
Not sure, see Description
```

### Description of issue

Hi. I'm not a programmer but I'd like to add the ability to use telnet to my Mac so I can access a MOO that I used way back in the early 1990s. My Mac is running macOS 13.2.1.

I followed the steps to install Homebrew on this page: https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/

And I got the "==> Installation successful!" message.

However, there seem to be a few extra steps and I can't figure them out. 

I also got this:

- Run these two commands in your terminal to add Homebrew to your PATH:
    (echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> /Users/skylamar/.zprofile
    eval "$(/opt/homebrew/bin/brew shellenv)"

I'm not that familiar with using Terminal and command-line commands. So I can't tell what the two separate commands are listed above. I don't know where each command begins and ends. 

So, could someone kindly tell me more clearly what the two commands are that I need to enter into Terminal? I would greatly appreciate it. I've made some guesses and they've been wrong.

Thanks,

Skylamar

**Comment by SMillerDev** [comment]
Each line is a separate command so:
```zsh
(echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> /Users/skylamar/.zprofile
```
and also
```zsh
eval "$(/opt/homebrew/bin/brew shellenv)"
```

**Reply by Skylamar** [reply]
Thanks so much! That worked.
